ecc/components/agenda-fieldset/agenda-fieldset.js

```diff
@@ -4,6 +4,11 @@ import { style } from './agenda-fieldset.css.js';
 
 const { LitElement, html, repeat } = await import(`${LIBS}/deps/lit-all.min.js`);
 
+const placeholderTitle = "Add agenda title";
+const titleMaxLengthText = "55 characters max";
```

Comment:
I like how we are moving away from word docs with these new features. Can we come up with a way to interpolate these numbers with the options property values?